### DISCUSSION TITLE
Piattaforma didattica

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#502] [Richieste di aiuto] [answered]
Salve, facendo un giro sul Vostro situo istituzionale ho notato una piattaforma: Apprendere.
Volevo sapere se fosse disponibile in rete e, in caso negativo, se si potesse ricevere una versione installabile...sarebbe utile per l'implementazione dell'interazione didattica.
Grazie mille.

**Comment by trinko** [comment, marked as answer]
La piattaforma didattica che abbiamo chiamato Apprendere non è altro che la notissima MOODLE, ottimo sistema di e-learning, open source e usatissimo: 
[sito MOODLE](https://moodle.org/)

**Reply by Lucaccc** [reply]
Grazie mille.
